Web/Edubase.Services/Enums/eLookupGroupType.cs

```diff
@@ -1,6 +1,9 @@
 
 namespace Edubase.Services.Enums
 {
+    /// <summary>
+    /// This has the same values as the database table EstablishmentGroupType so has a confusing name
+    /// </summary>
```

Comment:
Added comment to clarify this is establishment group type and not one other the other tables.